packages/zendesk-adapter/src/definitions/fetch/fetch.ts

```diff
@@ -1838,7 +1838,7 @@ export const createFetchDefinitions = (
       default: {
         resource: {
           serviceIDFields: ['id'],
-          onError: fetchUtils.errors.getInsufficientPermissionsError,
+          onError: fetchUtils.errors.createGetInsufficientPermissionsErrorFunction(),
```

Comment:
I think it should be 
```suggestion
          onError: fetchUtils.errors.createGetInsufficientPermissionsErrorFunction([403]),
```